src/python/map/pixels.py

```diff
@@ -472,6 +472,76 @@ def read_healpix_fits(self, path, comm_bytes=None):
         return
 
 
+    def broadcast_healpix_map(self, fdata, comm_bytes=None):
```

Comment:
@tskisner then inside `scan_signal` use this copy of `read_healpix_fits` that instead of reading from disk broadcasts the map I assembled on rank 0. Do you think this is fine?